miner/scroll_worker.go

```diff
@@ -858,7 +858,7 @@ func (w *worker) commit(force bool) (common.Hash, error) {
 
 	currentHeight := w.current.header.Number.Uint64()
 	maxReorgDepth := uint64(w.config.CCCMaxWorkers + 1)
-	if currentHeight > maxReorgDepth {
+	if !force && currentHeight > maxReorgDepth {
```

Comment:
Renaming `force` to `reorging` might make this function easier to understand.

Reply:
done